packages/backend-function/src/factory.test.ts

```diff
@@ -504,7 +510,7 @@ void describe('AmplifyFunctionFactory', () => {
       name: 'anotherName',
     });
     const functionStack = Stack.of(
-      functionFactory.getInstance(getInstanceProps).resources.lambda
+      functionFactory.getInstance(getInstanceProps).stack
     );
```

Comment:
nit: do we need `Stack.of` here?

Reply:
no, we don't, that would be a quick thing for me to fix since that looks like the only instance where I didn't properly remove `Stack.of`